### DISCUSSION TITLE
Authorisation error when accepting MRBase Data Access Agreement

### DISCUSSION THREAD

**Discussion by ghost** [MRCIEU/TwoSampleMR#363] [Q&A]
![image](https://user-images.githubusercontent.com/109031397/178237382-7d7974c3-9161-4c83-80bb-4f75adc26d51.png)

When I try to accept the data access agreement, I am redirected to this notice.
This prevents me from using the MRBase web app.

Any ways to get around this?

**Comment by remlapmot** [comment]
If this is still a problem could you try using the R code instead of the app. There is example R code in the vignettes listed on our website here

https://mrcieu.github.io/TwoSampleMR/articles/index.html

An analysis in an R script would be as follows (you need to change the exposure and outcome data for your research question).

```r
install.packages('remotes')
remotes::install_github('MRCIEU/TwoSampleMR')
library(TwoSampleMR)

bmi_exp_dat <- extract_instruments(outcomes = 'ieu-a-2')
chd_out_dat <- extract_outcome_data(snps = bmi_exp_dat$SNP, outcomes = 'ieu-a-7')
dat <- harmonise_data(bmi_exp_dat, chd_out_dat)

res <- mr(dat)

mr(dat, method_list = c("mr_egger_regression", "mr_ivw"))

mr_heterogeneity(dat)

mr_pleiotropy_test(dat)

p1 <- mr_scatter_plot(res, dat)
p1[[1]]

res_single <- mr_singlesnp(dat)

p2 <- mr_forest_plot(res_single)
p2[[1]]

p4 <- mr_funnel_plot(res_single)
p4[[1]]
```
